### PR TITLE
add step for delete the specifid ovs pod

### PR DESCRIPTION
**Pull request by zhaozhanqi**
logs:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/44981/consoleFull

@huiran0826 @anuragthehatter @weliang1
Could you help review, thanks

### CODE REVIEW

File: features/step_definitions/networking.rb [hunk 1]
```diff
@@ -683,3 +683,15 @@
   cb[cb_name] = @result[:response].chomp
   logger.info "The node's default interface is stored in the #{cb_name} clipboard."
 end
+
+Given /^I restart the ovs pod on the#{OPT_QUOTED} node$/ do | node_name |
+  ensure_admin_tagged
```
Comment (akostadinov) [inline features/step_definitions/networking.rb]: if this restart can affect other users of the cluster running stuff at the same time, then this needs to be `ensure_destructive_tagged`.

**Comment by akostadinov** [comment]
It is better, when you submit a new step, to also submit a scenario that uses it.

File: features/step_definitions/networking.rb (hunk 1 above)
Comment (zhaozhanqi) [reply features/step_definitions/networking.rb]: updated, thanks

**Comment by zhaozhanqi** [comment]
> It is better, when you submit a new step, to also submit a scenario that uses it.

yes, see https://github.com/openshift/cucushift/pull/7117

File: features/step_definitions/networking.rb (hunk 1 above)
Comment (anuragthehatter) [reply features/step_definitions/networking.rb]: /lgtm. yep, it seems to be an ensure_destructive_tagged

**Comment by akostadinov** [comment]
It is much preferable to submit at least one scenario in the same repo as an example. It doesn't matter tier of test.

**Pull request merged**